### PR TITLE
Address issue pointed out in CG call

### PR DESCRIPTION
**Pull request by bialpio**

### CODE REVIEW

File: hit-testing-explainer.md [hunk 1]
```diff
@@ -128,18 +156,19 @@ This is a partial IDL and is considered additive to the core IDL found in the ma
...
+  Promise<XRTransientInputHitTestSource> requestHitTestSourceForTransientInput(XRTransientInputHitTestOptionsInit options);
 };
 
 //
 // Frame
 //
 partial interface XRFrame {
-  FrozenArray<XRHitTestResult>? getHitTestResults(XRHitTestSource hitTestSource);
+  FrozenArray<XRHitTestResult> getHitTestResults(XRHitTestSource hitTestSource);
+  FrozenArray<XRTransientInputHitTestResult> getHitTestResultsForTransientInput(XRHitTestSource hitTestSource);
```
Comment (mounirlamouri) [inline hit-testing-explainer.md]: Should the parameter be `XRTransientInputHitTestSource`?

File: hit-testing-explainer.md [hunk 2]
```diff
@@ -151,10 +180,28 @@ dictionary XRTransientInputHitTestOptionsInit {
...
+
+//
+// Hit Test Results
+//
+
+[SecureContext, Exposed=Window]
+interface XRTransientInputHitTestResult {
+  XRInputSource inputSource;
+  FrozenArray<XRHitTestResult> results;
+};
```
Comment (mounirlamouri) [inline hit-testing-explainer.md]: nit: move below to keep order consistency?

File: hit-testing-explainer.md (hunk 1 above)
Comment (bialpio) [reply hit-testing-explainer.md]: Good catch, thanks!

File: hit-testing-explainer.md [hunk 3]
```diff
@@ -128,18 +156,19 @@ This is a partial IDL and is considered additive to the core IDL found in the ma
 //
 partial interface XRSession {
   Promise<XRHitTestSource> requestHitTestSource(XRHitTestOptionsInit options);
-  Promise<XRHitTestSource> requestHitTestSourceForTransientInput(XRTransientInputHitTestOptionsInit options);
+  Promise<XRTransientInputHitTestSource> requestHitTestSourceForTransientInput(XRTransientInputHitTestOptionsInit options);
```
Comment (Manishearth) [inline hit-testing-explainer.md]: `XRTransientInputHitTestOptionsInit` should be defined somewhere

Comment (bialpio) [reply hit-testing-explainer.md]: They are at line 149 (before PR) / 178 (after PR).

**Pull request merged**